<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserRegisterWgt</class>
 <widget class="QWidget" name="UserRegisterWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>公司名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_CompanyName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>电话：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Telephone"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_UserName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Pwd"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>真实姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_RealName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>权限：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_cbAuthority">
       <item>
        <property name="text">
         <string>管理员</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>操作员</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btn_Add">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
